### ISSUE TITLE
Using themes?

### ISSUE THREAD

**Issue by UplinkPhobia** [kxgames/glooey#16]
Hello,

I've been testing Glooey recently, and tried to use themes. Are they working? Since I didn't find anyting about them in the documentation or issues, I simply tried `from glooey.themes import golden` 
What I get is an error with resources not found for the kenney and golden themes, and an import error ("glooey" is not defined) for the wesnoth theme. 

```
[...]
    File "C:\Program Files (x86)\Python36-32\lib\site-packages\pyglet-1.3.0b1-py3.6.egg\pyglet\resource.py", line 474, in add_font
    File "C:\Program Files (x86)\Python36-32\lib\site-packages\pyglet-1.3.0b1-py3.6.egg\pyglet\resource.py", line 434, in file
pyglet.resource.ResourceNotFoundException: Resource "font/kenvector_future.ttf" was not found on the path.  Ensure that the filename has the correct captialisation.
```

```
>>> from glooey.themes import wesnoth
        File "C:\Program Files (x86)\Python36-32\lib\site-packages\glooey\themes\wesnoth.py", line 3, in <module>
            assets = glooey.themes.ResourceLoader('wesnoth')
    NameError: name 'glooey' is not defined
```

Is there something I'm doing wrong?

**Comment by kalekundert** [comment]
Sorry for being unresponsive; for some reason I wasn't getting email notifications for this project.  The wesnoth theme is not finished yet, but the others should work.  It looks like I might have forgotten to commit some files.  I'll check on that later today.

**Comment by kalekundert** [comment]
Again, really sorry for taking so long on this one.  I think the issue should be fixed by 7c2362d for the golden and kenney themes.  The wesnoth theme remains a work-in-progress.
